DOCS: Add more information about Layout C# inheritance (#2395)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/Documentation~/Layouts.md

* For layouts defined in JSON, you can specify the base layout in the `extends` property of the root node.
* For layouts created in code using [`InputControlLayout.Builder`](xref:UnityEngine.InputSystem.Layouts.InputControlLayout.Builder), you can specify a base layout using [`InputControlLayout.Builder.Extend()`](xref:UnityEngine.InputSystem.Layouts.InputControlLayout.Builder.Extend(System.String)).

> > **Note:** The layout hierarchy also determines which binding paths in Input Action Assets resolve controls on your device. When a path such as `<Gamepad>/buttonSouth` is resolved, the Input System walks the device's base layout chain upward until it finds a match. If your device class does not inherit from `Gamepad` (or whichever base layout the path references), that path will not resolves controls, no state change monitors will be created, and the corresponding actions will not fire. This also applies to custom base layouts: if you define a family of devices that share binding paths under a common layout name (e.g. `<MyBaseLayout>/...`), every device in that family must inherit from the C# class registered for that base layout.

Reviewer: The format for a note is 

> [!NOTE]
> The layout hierarchy.... 

etc

Reviewer: ```suggestion
> **Note:** The layout hierarchy also determines which binding paths in Input Action Assets resolve controls on your device. When a path such as `<Gamepad>/buttonSouth` is resolved, the Input System walks the device's base layout chain upward until it finds a match. If your device class doesn't inherit from `Gamepad` (or whichever base layout the path references), that path doesn't resolve controls, no state change monitors are created, and the corresponding actions don't fire. This also applies to custom base layouts: if you define a family of devices that share binding paths under a common layout name (such as`<MyBaseLayout>/...`), every device in that family must inherit from the C# class registered for that base layout.
```
